k8s/terrain.yml

```diff
@@ -36,38 +36,9 @@ spec:
         - name: signing-keys
           secret:
             secretName: signing-keys
-            items:
-              - key: private-key.pem
-                path: private-key.pem
-                mode: 0644
```

Comment:
It looks like we might be losing these explicit permission settings on each file - will that be an issue if so?

Reply:
That shouldn't be an issue. The file secret was being mounted in read-only mode before, so the write permissions weren't relevant anyway. Terrain also runs as root inside the container, so it should be able to read all of the files without any problem.